data/saas/config/aircall_config.yml

```diff
@@ -0,0 +1,109 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Aircall SaaS Config
+  type: aircall
+  description: A sample schema representing the Aircall connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: api.aircall.io
+    - name: api_token
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: api_key
+      configuration:
+        headers:
+          - name: Authorization
+            value: Basic <api_token>
+
+  test_request:
+    method: GET
+    path: /v1/contacts/
+    param_values:
+      - name: api_token
+        connector_param: api_token
+
+  endpoints:
+    - name: customer
+      requests:
+        read:
+          method: GET
+          path: /v1/users/<email>
+          data_path: user
+          param_values:
+            - name: email
+              identity: email
+            - name: api_token
+              connector_param: api_token
+    - name: contact
+      requests:
+        read:
+          method: GET
+          path: /v1/contacts/search
+          query_params:
+            - name: email
+              value: <email>
+          data_path: contacts
+          param_values:
+            - name: api_token
+              connector_param: api_token
+            - name: email
+              identity: email
+        update:
+          method: POST
+          path: /v1/contacts/<contact_id>
+          body: |
+            {
+               <masked_object_fields> 
+            }
+          param_values:
+            - name: contact_id
+              references:
+                - dataset: <instance_fides_key>
+                  field: contact.id
+                  direction: from
+            - name: api_token
+              connector_param: api_token
```

Comment:
remove update. We can only have an update or delete per endpoint.

Reply:
now removed update api